### PR TITLE
Add support for CONFIG_FILE_PATH/DOCKER_CONTEXT/DOCKER_FILE variables

### PR DESCRIPTION
**Pull request by AlexanderMoroz**
Can be very useful for non-standard docker builds (as we experienced)

### CODE REVIEW

File: .github/workflows/build-and-release-nhost-run.yaml [hunk 1]
```diff
@@ -16,6 +16,15 @@ on:
         type: string
         required: false
         default: .
+      DOCKER_FILE:
+        description: Path to the Dockerfile
+        type: string
+        required: false
```
Comment (dbarrosop) [inline .github/workflows/build-and-release-nhost-run.yaml:22]: should this have a default or is empty ok?

**Comment by dbarrosop** [comment]
Thanks!

File: .github/workflows/build-and-release-nhost-run.yaml (hunk 1 above)
Comment (dbarrosop) [reply .github/workflows/build-and-release-nhost-run.yaml:22]: I am mostly wondering if specifying it as empty will be treated as `""` and cause some issues. The consuming action says the `file` isn't required but not being required and being an empty string might be different things.

Comment (AlexanderMoroz) [reply .github/workflows/build-and-release-nhost-run.yaml:22]: From what I understand this is acceptable. But not sure 100%.  I can make it `./Dockerfile` by default

Comment (AlexanderMoroz) [reply .github/workflows/build-and-release-nhost-run.yaml:22]: Problem that by default it says {{context}}/Dockerfile, but I'm not sure that it's acceptable to use inputs in the inputs definition :P

Comment (dbarrosop) [reply .github/workflows/build-and-release-nhost-run.yaml:22]: yeah, let's make it "Dockerfile". That will probably translate to context/Dockerfile as the filepath should be relative to the context

Comment (AlexanderMoroz) [reply .github/workflows/build-and-release-nhost-run.yaml:22]: added @dbarrosop

**Comment by AlexanderMoroz** [comment]
feel free to merge if you ready

**Comment by dbarrosop** [comment]
Thanks a lot!

**Pull request merged**